packages/components/addon/components/hds/form/utils/getAriaDescribedBy.js

```diff
@@ -1,13 +1,16 @@
-import { ID_PREFIX as ERROR_ID_PREFIX } from '../error';
-import { ID_PREFIX as HELPER_TEXT_ID_PREFIX } from '../helper-text';
-
 export function getAriaDescribedBy(element) {
   let ariaDescribedBy = [];
-  ariaDescribedBy.push(`${HELPER_TEXT_ID_PREFIX}${element.id}`);
-  ariaDescribedBy.push(`${ERROR_ID_PREFIX}${element.id}`);
-  // append @ariaDescribedBy arg, if provided
-  if (element.args.ariaDescribedBy) {
-    ariaDescribedBy.push(element.args.ariaDescribedBy);
+
+  // append descriptor's IDs, if provided
+  if (element.descriptors) {
+    element.descriptors.forEach((descriptor) =>
+      ariaDescribedBy.push(descriptor)
+    );
+  }
```

Comment:
[nitpick] can be simplified even further, without looping on the entries
```suggestion
  if (element.descriptors) {
    ariaDescribedBy.concat(element.descriptors);
  }
```